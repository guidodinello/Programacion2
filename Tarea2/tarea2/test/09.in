# Implementar localizadorEnCadena, precedeEnCadena
localizadorEnCadena
insertarAlFinal (5,3.2)
localizadorEnCadena
finalCadena
localizadorEnCadena
siguiente
localizadorEnCadena
precedeEnCadena 1
finalCadena
precedeEnCadena 1
precedeEnCadena 0
precedeEnCadena 2
insertarAlFinal (5,3.4) 
insertarAlFinal (8,9.4) 
insertarAlFinal (9,2.6)
precedeEnCadena 1
precedeEnCadena 0
precedeEnCadena 2
siguiente
precedeEnCadena 1
precedeEnCadena 2
precedeEnCadena 3
finalCadena
precedeEnCadena 4
precedeEnCadena 5
anterior
precedeEnCadena 4
precedeEnCadena 3
Fin
