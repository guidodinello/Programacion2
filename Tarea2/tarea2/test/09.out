1>#  Implementar localizadorEnCadena, precedeEnCadena.
2>loc no pertenece a cad.
3>Insertado al final.
4>loc no pertenece a cad.
5>loc al final.
6>loc pertenece a cad.
7>loc al siguiente.
8>loc no pertenece a cad.
9>loc1 no precede a loc.
10>loc al final.
11>loc1 precede a loc.
12>loc1 no precede a loc.
13>loc1 no precede a loc.
14>Insertado al final.
15>Insertado al final.
16>Insertado al final.
17>loc1 precede a loc.
18>loc1 no precede a loc.
19>loc1 no precede a loc.
20>loc al siguiente.
21>loc1 precede a loc.
22>loc1 precede a loc.
23>loc1 no precede a loc.
24>loc al final.
25>loc1 precede a loc.
26>loc1 no precede a loc.
27>loc al anterior.
28>loc1 no precede a loc.
29>loc1 precede a loc.
30>Fin.
